support-task/src/test/java/test/camunda/SupportTaskServiceImpl.java

```diff
@@ -53,22 +53,19 @@ public String createSupportTask(String mainTaskInstanceId, String supportTaskTyp
         Task task = camundaUtils.waitForTask(processInstance.getId(), "supportTask");
 
         log.info("Support task created with id {}", task.getId());
-        return task.getId();
+        return processInstance.getId();
```

Comment:
Regardless of how a support **task** is created, it should return a **task** instance id